packages/typos-lsp/package.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: typos-lsp
+description: Source code spell checker
+homepage: https://github.com/tekumara/typos-vscode
+licenses:
+  - MIT
+  - Apache-2.0
```

Comment:
I can't see Apache-2.0 being referred to as their license anywhere

Reply:
Oh yeah that's totally a mistake, copy/paste error. My bad! Resolved in the latest commit